subsets/taxslim.obo

```diff
@@ -14,7 +15,8 @@ synonymtypedef: OMO:0003012 "acronym"
 synonymtypedef: scientific_name "scientific name"
 synonymtypedef: synonym "synonym"
 synonymtypedef: teleomorph "teleomorph"
-ontology: ncbitaxon/subsets/taxslim
+ontology: ncbitaxon/subsets/taxslim.obo
```

Comment:
Why did this become .obo?

Reply:
When I add the version IRI in the PR https://github.com/obophenotype/ncbitaxon/pull/112

Reply:
Ok, I guess I am fine with this, since you dont have a choice. In normal ODK pipelines, you would convert the OWL into OBO - here we do it the other way around, so the usual path, setting the iri in OWL, then converting, does not work.